internal/collect_es6_sources.bzl

```diff
@@ -29,15 +29,24 @@ def collect_es6_sources(ctx):
   """
 
   non_rerooted_files = depset()
+  rerooted_files = []
```

Comment:
don't move the declaration of this variable up, unless we actually reference it sooner